examples/loki/nurf_data.py

```diff
@@ -107,115 +107,163 @@ def nurf_file_creator(loki_file, path_to_loki_file, data):
         #print(list(hf['/entry/'].keys()))
         
         # create the various subgrous and their attributes
-
+        
         # UV subgroup
         grp_uv = hf.create_group("/entry/instrument/uv")
-        grp_uv.attrs["NX_class"] = 'NXdetector'
-        grp_uv.create_dataset('description', data='UV')
-        grp_uv.create_dataset('type',
-                              data='HR4PRO-UV-VIS-ES, DH-2000-FHS-DUV-TTL, QP600-2-SR')
+        grp_uv.attrs["NX_class"] = 'NXdetector_group'
+        
+        # subgroup for uv_spectra
+        uv_spectra=grp_uv.create_group("uv_spectra")
+        uv_spectra.attrs["NX_class"] = 'NXdetector'
+        uv_spectra_data=uv_spectra.create_dataset('data', data=data['UV_spectra'],
+                                           shape=data[
+                                               'UV_spectra'].shape,
+                                           dtype=np.int32)
+        uv_spectra_data.attrs['long_name']= 'uv_spectra'
+        uv_spectra_data.attrs['units']= 'a.u.'
```

Comment:
Using a unit like this will likely mean that things not to be fixed up downstream. Is this some sort of dimensionless intensity?

Reply:
Yes, I believe so. The ccd detectors of the spectrometers can be calibrated via a flat-field correction, but I don't think they are brought to absolut units or to any other unit. Should I remove the attribute for the unit?

Reply:
Consider using something that can be parsed/understood as "dimensionless"? I don't know if Nexus has a standardized name for this, but scipp understands, e.g., and empty string (`''`), `'1'`, or `'dimensionless'`.

Reply:
Generally the problem with AU is that when used, no meaningful number can be determined directly from the data.  It relies on 'other' information.  So it should be removed.

Reply:
I disagree with this assessment. It only shows that the detector is not calibrated, the user is given the correct information about the unit. If you wish to change this, I suggest to look into detector calibration.

Reply:
I don't think it's safe to make that assumption in general, and as Simon stated it is not an assumption SCIPP make.  AU is also the worst definition since it is a unit mentioned in the SI.

Reply:
What is your suggestion how to inform the user? To give no information by leaving it out?

Reply:
There is no unit supplied so there isn't anything to inform the user with.  Simon described the consequences downstream about supplying something v supplying nothing.  In fact, NXdata only aims to provide data in a given structure, without specifying how to plot, or the dimensionality (= shape) of said plot.  So, if a dataset has arbitrary units, units is superfluous information.  There are no units to specify, so there's no need to inform the user about anything?

I believe the definition has been updated since this PR was first created and it might provide  https://manual.nexusformat.org/classes/base_classes/NXdata.html .

As an additional comment which may be useful for people reading this in future, NX units should be compatible with UDUnits, [which has an entry for au but no mention of arbitrary units specified](https://docs.unidata.ucar.edu/udunits/current/udunits2-accepted.xml)

Reply:
My suggestion is to check out the scientific publications by the UV-Vis community. You will find multiple spectra on an absolute scale and spectra using a.u.